<launch>
    <node pkg="ftservoControl" type="control" name="control" output="screen" >
    <param name="id_up" type="int" value="4" />
    <param name="id_down" type="int" value="3" />
    </node>
</launch>
